tempest/api/workloadmgr/sanity/test_sanity_image_vol.py

```diff
@@ -0,0 +1,610 @@
+import sys
+import os
+import json
+import tempest
+import unicodedata
+import collections
+sys.path.append(os.getcwd())
+from tempest.api.workloadmgr import base
+from tempest import config
+from tempest import test
+from oslo_log import log as logging
+from tempest import tvaultconf
+from tempest import reporting
+import time
+from tempest import command_argument_string
+from tempest.util import cli_parser
+from tempest.util import query_data
+
+LOG = logging.getLogger(__name__)
+CONF = config.CONF
+
+class WorkloadTest(base.BaseWorkloadmgrTest):
+
+    credentials = ['primary']
+    workload_id = None
+    snapshot_id = None
+    vm_id = None
+    volume_id = None
+    floating_ip = None
+    kp = None
+    volumes = None
+    md5sums_dir_before = None
+    counter = None
+
+    @classmethod
+    def setup_clients(cls):
+        super(WorkloadTest, cls).setup_clients()
+        cls.client = cls.os.wlm_client
+        reporting.add_test_script(str(__name__))
+    
+    def assign_floating_ips(self, vm_id):
+        fip = self.get_floating_ips()
+        LOG.debug("\nAvailable floating ips are : \n".join(fip))
+        self.set_floating_ip(str(fip[0]),vm_id)
+        return(fip[0])        
+
+    def data_ops(self, flo_ip, mount_point, file_count):
+        global volumes
+        ssh = self.SshRemoteMachineConnectionWithRSAKey(str(flo_ip))
+        self.execute_command_disk_create(ssh, str(flo_ip), [volumes[0]], [mount_point])
+        self.execute_command_disk_mount(ssh, str(flo_ip), [volumes[0]], [mount_point])
+        self.addCustomfilesOnLinuxVM(ssh, mount_point, file_count)
+        ssh.close()
+
+    def calcmd5sum(self, flip, mount_point):
```

Comment:
There is already a method in base.py for the same. Can that be reused?

Reply:
yes i will reuse the existing one